nuget/Microsoft.Windows.CppWinRT.nuspec

```diff
@@ -13,7 +13,6 @@
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
     <license type="file">LICENSE</license>
     <projectUrl>https://github.com/Microsoft/xlang/tree/master/src/package/cppwinrt/nuget</projectUrl>
-    <iconUrl>https://aka.ms/cppwinrt.ico</iconUrl>
```

Comment:
Since iconUrl is deprecated, can you replace with an icon element pointing to vsix/cppwinrt.ico, and add that file to the nuspec?